modules/electromagnetics/test/tests/benchmarks/dipole_antenna/dipole.i

```diff
@@ -0,0 +1,102 @@
+# Verification Benchmark - Half-wave Dipole Antenna (Frequency Domain)
+# Resonant Frequency = 1 GHz
+# Wave Propagation Medium: Vacuum
+
+[Mesh]
+  [./file_mesh]
+    type = FileMeshGenerator
+    file = dipole_antenna_1G.msh
+  [../]
+  [./refine]
+    type = RefineBlockGenerator
+    input = file_mesh
+    block = 'vacuum'
+    refinement = 2
+  [../]
+[]
+
+[Variables]
+  [./E_real]
+    order = FIRST
+    family = NEDELEC_ONE
+  [../]
+  [./E_imag]
+    order = FIRST
+    family = NEDELEC_ONE
+  [../]
+[]
+
+[Kernels]
+  [./curl_curl_real]
+    type = CurlCurlField
+    variable = E_real
+  [../]
+  [./coeff_real]
+    type = VectorCoeffField
+    variable = E_real
+    func = '-(2*pi*1e9/3e8)*(2*pi*1e9/3e8)'  # -(wave number)^2
+  [../]
+  [./curl_curl_imag]
+    type = CurlCurlField
+    variable = E_imag
+  [../]
+  [./coeff_imag]
+    type = VectorCoeffField
+    variable = E_imag
+    func = '-(2*pi*1e9/3e8)*(2*pi*1e9/3e8)'  # -(wave number)^2
+  [../]
+[]
+
+[BCs]
+  [./antenna_real]                #TODO: Replace with proper antenna surface current condition
```

Comment:
Maybe remove the TODO?

Reply:
Should probably just make a proper issue for this one instead of the TODO. This was just an easily searchable reminder for me.